builder/postaggregation/quantiles_doubles_sketch_to_quantile.go

```diff
@@ -4,18 +4,11 @@ package postaggregation
 // PostAggregator section in https://druid.apache.org/docs/latest/development/extensions-core/datasketches-quantiles.html#quantile
 type QuantilesDoublesSketchToQuantile struct {
 	Base
-	Field    *QuantilesDoublesSketchToQuantileField `json:"field,omitempty"`
-	Fraction float64                                `json:"fraction,omitempty"`
+	Field    *QuantilesDoublesSketchField `json:"field,omitempty"`
```

Comment:
this is to set the field name generic as per https://druid.apache.org/docs/latest/development/extensions-core/datasketches-quantiles.html